packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1056,6 +1056,12 @@ export default class Meeting extends StatelessWebexPlugin {
         });
       }
     });
+
+    this.locusInfo.on(LOCUSINFO.EVENTS.MEDIA_INACTIVITY, () => {
+      this.reconnect();
```

Comment:
Do we need metrics for when we receive this event?

Reply:
Its better to have one not sure if edonus sends one to the call-analyzer

Reply:
Ok, we can investigate if we need one, not needed in this PR though.